### PR TITLE
fix: padding between label and tooltip icon on transaction review screen

### LINKED ISSUE #4693: Fix padding between label and tooltip icon on transaction review screen
Screenshot 2023-12-13 at 18 12 32

### PR DESCRIPTION
**Pull request by Nithishvb**
Closes #4693

This PR addresses and resolves the alignment issue with the fees warning on the transaction review screen. 

Screenshot 2023-12-18 212757

### CODE REVIEW

**Review by pete-watters — CHANGES_REQUESTED** [review]
Thanks for all your help @Nithishvb 

This change looks good to me but before I can merge it we need to clean up the branch. 

We `rebase` our branches to keep a clean history and don't use merge commits. 

I can approve and merge this if you do one of these things:

1. update this branch via rebase
2. create this change in a fresh branch off `dev` 
3. use interactive rebase to remove the merge commits

Option 1. is probably the easiest but I'm not sure it will work if you have already done a merge commit. 

You can do it on the command line with:
```
git rebase origin/dev
git push -f
```
Or in the UI here: 
Screenshot 2023-12-22 at 09 03 48

**Comment by Nithishvb** [comment]
Hi @pete-watters, Thank you for your review and feedback! I appreciate your guidance on maintaining a clean history.
I will proceed with option 2 to address the concerns. I will create this change in a fresh branch off dev and submitting a fresh pull request.

**Pull request closed without merging**
